### ISSUE TITLE
actually doing the training for the basic control

### ISSUE THREAD

**Issue by yamins81** [dicarlolab/ThreeDWorld#65]
What is the loss function that we'll be using for the control?   How will we handle multiple labels?  Softmax matching?

**Comment by yamins81** [comment]
(1) figure out that model
(2) figuring out the loss function
(3) putting it into tfutils script
